monitoring/v1alpha1/register.go

```diff
@@ -0,0 +1,41 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName     = "monitoring.openshift.io"
```

Comment:
who is the monitoring team lead these days?  I'd like their ack on taking their name.

Reply:
That would be me, please consider this acked.